Cython/Utility/Coroutine.c

```diff
@@ -499,6 +499,9 @@ static int __pyx_Generator_init(PyObject *module); /*proto*/
 //@requires: ModuleSetupCode.c::IncludeStructmemberH
 
 #include <frameobject.h>
+#if PY_VERSION_HEX >= 0x030b00a5
```

Comment:
Technically, the include will only be needed since 0x030b00a6 but it's fine to include it in Python 3.11 alpha 5. I suggest to define the Py_BUILD_CORE macro to include an internal C API header file. Right now, it's not needed, but I consider requiring it later.

```
#  ifndef Py_BUILD_CORE
#    define Py_BUILD_CORE 1
#  endif
```

Same remarks for the other modified files.

Reply:
Fair, but then it would not work for me locally.  What is the best way to test this sort of thing in the "interstitial" time?  I guess I could just locally patch my CPython source to be the next release prematurely?

Should I also include the `!CYTHON_COMPILING_IN_PYSTON ` check everywhere?

Reply:
3.11a6 will hopefully be out quite soon. Maybe faster than this gets merged. Not sure when it will become available on Github Actions, though.